wsd/COOLWSD.cpp

```diff
@@ -5962,29 +5962,6 @@ int COOLWSD::innerMain()
     Util::getVersionInfo(version, hash);
     LOG_INF("Coolwsd version details: " << version << " - " << hash << " - id " << Util::getProcessIdentifier() << " - on " << Util::getLinuxVersion());
 
-    std::size_t availableMemoryMb = Util::getTotalSystemMemoryKb()/1024;
```

Comment:
`Admin` is initialized before we get here (it's static/singleton). So this was generating duplicate logs.